### PR TITLE
more consistent class prefixing

### PR DESCRIPTION
**Pull request by jbwyme**

### CODE REVIEW

File: lib/components/modals/modal.jade [hunk 1]
```diff
@@ -1,24 +1,24 @@
-.mp-modal(class={
-  alert: $component.isAttributeEnabled('alert'),
-  absolute: $component.isAttributeEnabled('not-fullscreen'),
-  closed: $component.isAttributeEnabled('closed'),
+.mp-modal-stage(class={
```
Comment (ilyakamens) [inline lib/components/modals/modal.jade]: why `stage`?

Comment (ilyakamens) [inline lib/components/modals/modal.jade]: you know i hate this word can we use a different one

**Comment by ilyakamens** [comment]
cool LGTM%C

File: lib/components/modals/modal.jade (hunk 1 above)
Comment (jbwyme) [inline lib/components/modals/modal.jade]: if you want to suggest a better name go for it, take a look at the class hierarchy first though

**Comment by tdumitrescu** [comment]
Shipitsquirrel

File: lib/components/modals/modal.jade (hunk 1 above)
Comment (ilyakamens) [inline lib/components/modals/modal.jade]: it's hard for me to suggest a better name because i don't understand what stage is

**Pull request merged**